src/tir/ir/stmt_functor.cc

```diff
@@ -700,10 +700,11 @@ Array<Range> Substitute(const Array<Range>& region, const Map<Var, PrimExpr>& vm
 }
 
 void PreOrderVisit(const ObjectRef& stmt_or_expr,
-                   const std::function<bool(const ObjectRef&)>& fvisit) {
+                   const std::function<bool(const ObjectRef&)>& fvisit, bool visit_init_block) {
```

Comment:
I think it makes no sense to add a parameter in the base class to determine if we visit the init block. Write an additional visitor may be a better choice.

Reply:
Yeah the param has no usage after confirmation. Just removed it.